images/linux/scripts/installers/pulumi.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+################################################################################
+##  File:  pulumi.sh
+##  Desc:  Installs Pulumi
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/install.sh
+
+# Install Pulumi
+VERSION=$(curl --fail --silent -L "https://www.pulumi.com/latest-version")
+TARBALL_URL="https://get.pulumi.com/releases/sdk/pulumi-v${VERSION}-linux-x64.tar.gz"
```

Comment:
@cnunciato are you with Pulumi? does this look like a reliable way to get the latest?

Reply:
This way is provided by official install process so it should be reliable enough

Reply:
@alepauly I am, and yes, that's the way.